db/schema.rb

```diff
@@ -10,9 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2017_06_26_055158) do
+ActiveRecord::Schema.define(version: 2021_12_05_182329) do
 
-  create_table "credence_answers", id: :integer, force: :cascade do |t|
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "credence_answers", id: :serial, force: :cascade do |t|
```

Comment:
The default for Rails 5.2 is now `:serial`, rather than `:integer`. I can back these out if anyone is concerned about how this might impact the production database.

Reply:
Seems fine